Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -25,10 +25,11 @@
 use Symfony\CS\Error\Error;
 use Symfony\CS\Error\ErrorsManager;
 use Symfony\CS\Fixer;
+use Symfony\CS\FixerFactory;
 use Symfony\CS\FixerInterface;
 use Symfony\CS\Linter\Linter;
 use Symfony\CS\Linter\UnavailableLinterException;
-use Symfony\CS\Utils;
+use Symfony\CS\RulesSet;
```

Comment:
It may be a good point, maybe even [Ruleset](http://www.yourdictionary.com/ruleset) ?